### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by TiagoCavalcante**
New features were added.

### CODE REVIEW

**Comment by TiagoCavalcante** [comment]
@axkr I have added a page for primitives.

File: docs/primitives/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+For example, a 3d plot can be converted into [polygons](polygon).
+
+If you are using this library as a backend to a Wolfram-Language-like project
+that supports [Plot3D](https://reference.wolfram.com/language/ref/Plot3D.html)
+you may convert that to a
+[Graphics3D](https://reference.wolfram.com/language/ref/Graphics3D.html)
+object (this library goal is to fully support it), or create a conversion
+function for every plot function.
+
+An *example* of conversion code for a
```
Comment (rocky) [inline docs/primitives/index.md:11]: I don't understand the italic here.

**Comment by rocky** [comment]
In the top-level index.md:

> mathics-threejs-backend is a JavaScript library for rendering Mathics (and eventually Wolfram Language) Graphics3D objects.

I think it okay to mention that Symja here if this is using it.

File: docs/primitives/index.md (hunk 1 above)
Comment (TiagoCavalcante) [reply docs/primitives/index.md:11]: In the current version of mathics-threejs-backend you need to invert the y and z coordinates to the plot to be "horizontal". But I plan to fix that next year.

**Pull request merged**